modules/gke-cluster-standard/variables.tf

```diff
@@ -52,6 +63,11 @@ variable "cluster_autoscaling" {
       min = number
       max = number
     }))
+    gpu_resources = optional(list(object({
+      resource_type = string
+      min           = number
+      max           = number
```

Comment:
This is inconsistent with the two othe limits we already implement (cpu/mem). You probably want to skip the `resource_type` key and just hard-code it in the block if the users provides a value for `gpu_resources`

Reply:
Done in e2c030d8f7521f4f478a0058f7e9f40f69ca7a67 :slightly_smiling_face:

Reply:
Well for GPU, there can be several `resourceType` such as `nvidia-tesla-k80` or  `nvidia-tesla-v100`.
There's a long list of GPU types so I think it's better to let a user choose which one to use.

Ref:
 - https://cloud.google.com/kubernetes-engine/docs/how-to/node-auto-provisioning#gpu_limits

Reply:
I see. We can leave like this for now but I think eventually we'll want to use a single field for all limits.